examples/experimental/onnx/performance_checker.py

```diff
@@ -29,6 +29,7 @@
 from nncf.experimental.onnx.engine import ONNXEngine
 from nncf.experimental.onnx.samplers import create_onnx_sampler
 from nncf.experimental.post_training.api.dataset import Dataset
+from examples.experimental.onnx.accuracy_checker import MSCocoSegmentationToVOCConverter  # pylint: disable=unused-import
```

Comment:
This is because `MSCocoSegmentationToVOCConverter` is a custom converter not belonging to `accuracy_checker` by default. So, this should be imported to register it to the factory constructor at runtime.

Reply:
Why do you really need writing your own provider? Does not AC support VOC dataset?

Reply:
AC supports VOC dataset. But, it is required to use COCO dataset with the only classes existed in the VOC datasets (e.g. human, car, etc...). I don't know whether it is a bug of [`MSCocoSegmentationConverter`](https://github.com/openvinotoolkit/open_model_zoo/blob/dc392048fc72f26dd23be33140f1fda7e4200c39/tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/ms_coco.py#L438-L448) or originally not supported. Anyway, I had to make `MSCocoSegmentationToVOCConverter` to resolve it.